lib/galaxy/tools/__init__.py

```diff
@@ -2352,8 +2347,8 @@ def to_json (self, trans, kwd={}, is_workflow=False):
         tool_message = ''
         if job:
             try:
-                job_params = job.get_param_values( trans.app, ignore_errors = True )
-                job_messages = self.check_and_update_param_values( job_params, trans, update_values=False )
+                job_params = job.get_param_values( trans.app, ignore_errors=True )
+                self.check_and_update_param_values( job_params, trans, update_values=False )
```

Comment:
This line originates from: https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/webapps/galaxy/controllers/tool_runner.py#L204. So far it seemed that we do not need these messages in the new structure. Previously they were parsed as 'old_errors' to handle inputs. See also: https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/tools/__init__.py#L1623.